db/migrate/20220103063359_create_articles.rb

```diff
@@ -1,7 +1,7 @@
 class CreateArticles < ActiveRecord::Migration[6.1]
   def change
     create_table :articles do |t|
-      t.string :title
+      t.text :title
```

Comment:
change to `text` because this data type is variable and is of unlimited length.

Reply:
why is text an unlimited length? do you mean that you expect the title to have 10,000 characters? 1 mil?

Reply:
because you do not need to specify the length limit (`n`) in `text`, unlike `char` or `varchar` where the storage is allocated with `n` characters or bytes, so one must not exceed `n` when inserting values into that column. 

`text` == `varchar` without `n` , use either one also can.